$OpenBSD: patch-meson_build,v 1.1 2019/01/01 11:36:09 ajacoutot Exp $

XXX https://gitlab.gnome.org/GNOME/dconf/issues/47
(the fix/MR does not work, it makes dconf SEGFAULT)

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -82,6 +82,5 @@ subdir('gsettings')
 subdir('client')
 subdir('bin')
 subdir('docs')
-subdir('tests')
 
 meson.add_install_script('meson_post_install.py', gio_module_dir)
